doc/setup/email_gateway.rst

```diff
@@ -0,0 +1,40 @@
+:banner: banners/email_gateway.jpg
+
+==================
+Odoo email gateway
+==================
+
+The Odoo mail gateway is a better alternative than using an "incoming mail server"
+because the messages are received instantaneously.
+
+The principle is straightforward. Your incoming SMTP server will receive emails and directly forward
+them through a script (``odoo/addons/mail/static/scripts/odoo-mailgate.py``).
+
+This script will simply connect to your database through XMLRPC to send the mails to MailThread.message_process().
+
+Prerequisites
+-------------
+- Administrator access to the database
+- Technical knowledge on how to configure a email server
+- Your own mail server such as Postfix or Exim
+
+For Postfix server
+------------------
+In you alias config (/etc/aliases):
+
+.. code-block:: text
+email@address: "|/odoo-directory/addons/mail/static/scripts/odoo-mailgate.py -d <database-name> -u <userid> -p <password>"
+.. note:: Ressources
+    More details in the following ressources:
+    - `Postfix <http://www.postfix.org/documentation.html>`
+    - `Postfix aliases <http://www.postfix.org/aliases.5.html>`
+    - `Postfix virtual <http://www.postfix.org/virtual.8.html>`
+
+
+For Exim server
+---------------
+.. code-block:: text
+ *: |/odoo-directory/addons/mail/static/scripts/odoo-mailgate.py -d <database-name> -u <userid> -p <password>
+.. note:: Ressources
+    More details in the following ressources:
+    - `Exim <https://www.exim.org/docs.html>`
```

Comment:
I would add a paragraph
"if you don't have access/manage your email server " + links to the configuration of Incoming email servers